### DISCUSSION TITLE
How do I center the markdown

### DISCUSSION THREAD

**Discussion by ARR4NN** [remarkjs/remark#1127] [Q&A] [answered]
I am trying to center the markdown but nothing is working for me
![image](https://user-images.githubusercontent.com/70060472/224345032-5cb608dd-cf95-46f5-88ba-e6809ef8b9cc.png)
Everything in the black box is markdown the images and video iframe.  
![image](https://user-images.githubusercontent.com/70060472/224345197-921ff718-24d5-4cfb-9056-bdfc339aeb28.png)

**Comment by ChristianMurphy** [comment, marked as answer]
Welcome @ARR4NN!
`text-align: center` can be applied to content to center it.
Runnable example: https://codesandbox.io/s/sweet-jasper-hdbgb3?file=%2Fsrc%2Fapp.tsx
Most likely there is an issue with your CSS-in-JS (styled components) setup, causing the styles to not apply.
